extensions/wikia/AdMixExperiment/scripts/AdMixExperiment.js

```diff
@@ -267,12 +281,18 @@ require(['ext.wikia.adEngine.adContext', 'wikia.abTest', 'wikia.throttle'], func
 
 		function onRightRailReady() {
 			$recircWrapper = $('#recirculation-rail');
+			$adAndRecircWrapper = $('#WikiaAdInContentPlaceHolder');
+
+			if (recircEnabled) {
+				$adAndRecircWrapper.addClass('ad-mix-experiment-enabled');
+			}
 
 			// There is no recirculation right rail module on non-en wikis
 			if (window.wgContentLanguage === 'en') {
 				$recircWrapper.one('premiumRecirculationRail.ready', onRecircReady);
 			} else {
-				recircEnabled = false;
+				// TODO check which experiment needs it
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)